package.json

```diff
@@ -4,7 +4,9 @@
   "description": "",
   "main": "index.js",
   "scripts": {
-    "dev": "node api/index.js", 
+    "dev": "nodemon index.js",
```

Comment:
I added `nodemon` and changed the file we're using to start our app.
nodemon is a module which watches all your files and restarts our ssb/api if anything changes. Saves us having to stop + start it all the time

Reply:
THANK YOOOUU!!!